docs/dev-guide/integration/rest-api/content/associate.rst

```diff
@@ -40,7 +40,6 @@ with an importer that supports the type of units being copied.
       }
     },
    'override_config': {
-     'resolve_dependencies: true,
```

Comment:
The API endpoint resolve_dependencies was already removed in some other PR. It was unused, undocumented and how this line got into the doc example i have no idea. You can check this issue for more info https://pulp.plan.io/issues/812 
In this PR i remove the remainings of resolve_dependencies like manager and tests.